src/features/metadata-based-view/__tests__/MetadataBasedItemList.test.js

```diff
@@ -179,16 +200,20 @@ describe('features/metadata-based-view/MetadataBasedItemList', () => {
 
     describe('getGridCellData(columnIndex, rowIndex)', () => {
         test.each`
-            columnIndex | rowIndex | cellData
-            ${0}        | ${1}     | ${pdfIcon}
-            ${1}        | ${1}     | ${pdfNameButton}
-            ${2}        | ${1}     | ${'bill'}
-            ${3}        | ${1}     | ${100.12}
-            ${0}        | ${2}     | ${mp4Icon}
-            ${1}        | ${2}     | ${mp4NameButton}
-            ${2}        | ${2}     | ${'receipt'}
-            ${3}        | ${2}     | ${200.88}
-        `('cellData for row: $rowIndex, column: $columnIndex', ({ columnIndex, rowIndex, cellData }) => {
+            columnIndex | rowIndex | cellData         | dataType
+            ${0}        | ${1}     | ${pdfIcon}       | ${undefined}
+            ${1}        | ${1}     | ${pdfNameButton} | ${undefined}
+            ${2}        | ${1}     | ${'bill'}        | ${'string'}
+            ${3}        | ${1}     | ${100.12}        | ${'float'}
+            ${4}        | ${1}     | ${createdAt}     | ${'date'}
+            ${5}        | ${1}     | ${undefined}     | ${undefined}
+            ${0}        | ${2}     | ${mp4Icon}       | ${undefined}
+            ${1}        | ${2}     | ${mp4NameButton} | ${undefined}
+            ${2}        | ${2}     | ${'receipt'}     | ${'string'}
+            ${3}        | ${2}     | ${200.88}        | ${'float'}
+            ${4}        | ${2}     | ${createdAt}     | ${'date'}
+            ${5}        | ${2}     | ${undefined}     | ${undefined}
```

Comment:
Do numerical values in this view always have the `float` data type? Or could they be integers?

Reply:
These are data types of Box metadata fields: https://developer.box.com/guides/metadata/fields/